PWGPP/AliOfflineTrigger.cxx

```diff
@@ -639,15 +641,20 @@ void   AliOfflineTrigger::ExtractSelected(const char *rawList, const char * trig
   */
   TObjArray* rawArray = 0;
   if (TPRegexp(".xml$").Match(rawList)){
+#ifdef WITHALIEN
     TAlienCollection *coll = (TAlienCollection *)TAlienCollection::Open(rawList);
     Int_t nFiles =  coll->GetNofGroups();
     rawArray=new TObjArray(nFiles);
     while( coll->Next()){
       rawArray->AddLast(new TObjString(coll->GetTURL()));
     }
+#endif
   }else{
     rawArray = gSystem->GetFromPipe(TString::Format("cat %s", rawList)).Tokenize("\n");  
   }
+  if (rawArray == 0) {
+    ::Error("AliOfflineTrigger::ExtractSelected","No AliEn support compiled-in");
```

Comment:
I'd raise a fatal error here, and maybe move it into an `#ifdef ... #else ... #endif` block.